js/bit4you.js

```diff
@@ -0,0 +1,624 @@
+                'spot': true,
+                'margin': undefined,
+                'swap': undefined,
+                'future': undefined,
+                'option': undefined,
+                'cancelOrder': true,
+                'createOrder': true,
+                'fetchBalance': true,
+                'fetchClosedOrders': true,
+                'fetchCurrencies': true,
+                'fetchMarkets': true,
+                'fetchMarkOHLCV': false,
+                'fetchOHLCV': true,
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrderBook': true,
+                'fetchStatus': true,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTrades': false,
+                'fetchTransactions': true,
+                'withdraw': true,
+            },
+            'api': {
+                'public': {
+                    'post': [
+                        'market/orderbook',
+                    ],
+                    'get': [
+                        'market/assets',
+                        'market/list',
+                        'market/summaries',
+                        'udf/history', // ?iso={iso}&resolution={resolution}&from={from}&to={to}
+                    ],
+                },
+                'private': {
+                    'post': [
+                        'wallet/balances',
+                        'wallet/blockchain-history',
+                        'wallet/send',
+                        'portfolio/history',
+                        'order/pending',
+                        'order/info',
+                        'order/list',
+                        'order/create',
+                        'order/cancel',
+                    ],
+                    'get': [],
+                },
+            },
+            'timeframes': {
+                '1m': '1m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '4h': '4h',
+                '1d': '1d',
+            },
+            'urls': {
+                'logo': 'https://www.bit4you.io/img/logo/logo.svg',
+                'www': 'https://www.bit4you.io/',
+                'doc': 'https://docs.bit4you.io/',
+                'fees': 'https://www.bit4you.io/services#fees',
+                'api': {
+                    'coinmarketcap': 'https://www.bit4you.io/api/cmc/v1/', // ex:"/api/cmc/v1/summary"
+                    'graphql': 'https://www.bit4you.io/api/markets/graphql',
+                    'url': 'https://www.bit4you.io/api',
+                },
+            },
+        });
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1h', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        // console.log (market)
+        const request = {
+            'symbol': market['symbol'].replace ('/', '-'),
```

Comment:
Whenever we need to send the symbol to the exchange we use `market['id']` (as I said earlier id stores the exchange specific representation)

Reply:
market['id'] return  the symbol without delimiter and we need the delimiter '-' in our api for the symbol. It's why I use market['symbol'] that return '/' as delimiter that we replace by '-'